docs/bugs.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "uid": "SOL-2022-6",
+        "name": "CorruptABIEncodingWithYul",
+        "summary": "Encoding a static calldata array followed by a dynamic array results in a corrupt length value for the dynamic array.",
+        "description": "When ABI-encoding a static calldata array, we used to clean the last 32 byte slot after the array data by overwriting it with ``0``. This is problematic if the data of the dynamic array was already written there. This is the case when encoding a dynamic array followed by a static calldata array. Then the length of the dynamic array is overwritten.",
+        "introduced": "0.5.8",
+        "fixed": "0.8.16",
+        "severity": "low"
```

Comment:
I felt it was `low` because I thought: The worst effect is that the length of the array that was sent will be 0. The sender would just send corrupt data like that in the first place, so we're not really worse off than before in terms of "terrible things the sender can do"

Reply:
ok, you have a point. But I think the potential is higher for accidental data loss than for something malicious. Sending empty array when it was meant to contain stuff can still easily break things. We still include that in the severity assessment, right?